### PR TITLE
Updated titles to reflect style guide standards for AAP books

### PR DESCRIPTION
**Pull request by annibond**
Updated remaining titles for [AAP-17967](https://issues.redhat.com/browse/AAP-17967) for consistency/style standards.

### CODE REVIEW

File: downstream/titles/automation-mesh/master.adoc [hunk 1]
```diff
@@ -9,12 +9,12 @@ include::attributes/attributes.adoc[]
 
 
 // Book Title
-= Red Hat Ansible Automation Platform Automation Mesh Guide for VM-based installations
+= Red Hat Ansible Automation Platform Automation mesh guide for VM-based installations
```
Comment (dcdacosta) [inline downstream/titles/automation-mesh/master.adoc:12]: I believe Automation should be lower case too, so:
Red Hat Ansible Automation Platform automation mesh guide for VM-based installations

File: downstream/titles/automation-mesh/docinfo.xml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<title>Red Hat Ansible Automation Platform Automation Mesh Guide for VM-based installations</title>
+<title>Red Hat Ansible Automation Platform Automation mesh guide for VM-based installations</title>
```
Comment (dcdacosta) [inline downstream/titles/automation-mesh/docinfo.xml:1]: I believe Automation should be lower case too, so:
Red Hat Ansible Automation Platform automation mesh guide for VM-based installations

File: downstream/titles/controller/controller-admin-guide/docinfo.xml [hunk 3]
```diff
@@ -1,4 +1,4 @@
-<title>Automation Controller Administration Guide</title>
+<title>Automation Controller administration guide</title>
```
Comment (dcdacosta) [inline downstream/titles/controller/controller-admin-guide/docinfo.xml:1]: Automation Controller is not a proper noun so this should be:

Automation controller administration guide

File: downstream/titles/controller/controller-admin-guide/master.adoc [hunk 4]
```diff
@@ -9,10 +9,10 @@ include::attributes/attributes.adoc[]
 
 
 // Book Title
-= Automation Controller Administration Guide
+= Automation Controller administration guide
```
Comment (dcdacosta) [inline downstream/titles/controller/controller-admin-guide/master.adoc:12]: Automation Controller is not a proper noun so this should be:

Automation controller administration guide

File: downstream/titles/controller/controller-user-guide/docinfo.xml [hunk 5]
```diff
@@ -1,4 +1,4 @@
-<title>Automation Controller User Guide</title>
+<title>Automation Controller user guide</title>
```
Comment (dcdacosta) [inline downstream/titles/controller/controller-user-guide/docinfo.xml:1]: Automation Controller is not a proper noun so this should be:

Automation controller administration guide

File: downstream/titles/controller/controller-user-guide/master.adoc [hunk 6]
```diff
@@ -9,7 +9,7 @@ include::attributes/attributes.adoc[]
 
 
 // Book Title
-= Automation Controller User Guide
+= Automation Controller user guide
```
Comment (dcdacosta) [inline downstream/titles/controller/controller-user-guide/master.adoc:12]: Automation Controller is not a proper noun so this should be:

Automation controller administration guide

File: downstream/titles/dev-guide/master.adoc [hunk 7]
```diff
@@ -7,7 +7,7 @@ include::attributes/attributes.adoc[]
 
 
 // Book Title
-= {PlatformName} Creator Guide
+= Ansible Automation Platform creator guide
```
Comment (dcdacosta) [inline downstream/titles/dev-guide/master.adoc:10]: This needs to match what is in the docinfo.xml file so you should add Red Hat to the title here

Red Hat Ansible Automation Platform creator guide

File: downstream/titles/eda/eda-getting-started-guide/docinfo.xml [hunk 8]
```diff
@@ -1,4 +1,4 @@
-<title>Getting Started with Event-Driven Ansible Guide</title>
+<title>Getting started with event-driven Ansible guide</title>
```
Comment (dcdacosta) [inline downstream/titles/eda/eda-getting-started-guide/docinfo.xml:1]: Event-Driven Ansible is a proper noun and should be capitalized

Getting Started with Event-Driven Ansible guide

File: downstream/titles/eda/eda-getting-started-guide/master.adoc [hunk 9]
```diff
@@ -9,7 +9,7 @@ include::attributes/attributes.adoc[]
 
 
 // Book Title
-= Getting Started with Event-Driven Ansible Guide
+= Getting started with event-driven Ansible guide
```
Comment (dcdacosta) [inline downstream/titles/eda/eda-getting-started-guide/master.adoc:12]: Event-Driven Ansible is a proper noun and should be capitalized

Getting Started with Event-Driven Ansible guide

**Review by dcdacosta — CHANGES_REQUESTED** [review]
Hi @annibond ,

There are a few corrections needed, then this should be good to go.

~ Donna

**Comment by annibond** [comment]
@dcdacosta I have made those changes

**Review by dcdacosta — APPROVED** [review]
LGTM! Nice job!

**Pull request merged**